pkg/node/address.go

```diff
@@ -233,6 +233,14 @@ func SetInternalIPv4Router(ip net.IP) {
 	})
 }
 
+// GetInternalIPv6Router returns the cilium internal IPv6 node address. This must not be conflated with
```

Comment:
nit: I would move the function with the other IPV6 function [between the two following functions](https://github.com/cilium/cilium/blob/fedab8406135904ded866d0dbff2bd9d763e8686/pkg/node/address.go#L332).
```
// GetIPv6 returns the IPv6 address of the node
func GetIPv6() net.IP {
	n := getLocalNode()
	return clone(n.GetNodeIP(true))
}

// GetIPv6Router returns the IPv6 address of the router, e.g. address
// of cilium_host device.
func GetIPv6Router() net.IP {
	n := getLocalNode()
	return clone(n.GetCiliumInternalIP(true))
}
```